ipatests/test_webui/data_user.py

```diff
@@ -85,3 +138,105 @@
         ('combobox', 'gidnumber', '77777'),
     ]
 }
+
+PKEY5 = '1spe.cial_us-er$'
+DATA5 = {
```

Comment:
I'm beginning to think whether we should stop using general `DATAX` names for data which serve for a specific test. Originally there was only a need for more records without a special meaning to test the simplest CRUD or add more records as members.

Here if somebody changes the data in future, we could easily miss its purpose - a test case. So e.g. instead of DATA, it can be DATA_SPECIAL_CHARS. Same with other DATAs.